### ISSUE TITLE
Updating memberships to remove post_id returns error

### ISSUE THREAD

**Issue by kaerumy** [Sinar/popit_ng#252]
```
{"errors":{"post_id":["This field may not be null."]}}
```

Test membership:

https://api.popit.sinarproject.org/en/memberships/545e60e45222837c2c0599e0

payload:

```
{ 'post_id' : null }

**Comment by sweemeng** [comment]
from popit/serializer/memberships.pyy
```
post_id = CharField(max_length=255, required=False)
```

we need to add allow_null parameter like

```
death_date = CharField(allow_null=True, default=None, allow_blank=True)
```

**Comment by kaerumy** [comment]
Maybe not:

https://api.popit.sinarproject.org/en/memberships/545e61995222837c2c05a251

payload = { 'post_id': None }

returns:

```'{"result":{"id":"545e61995222837c2c05a251","person":{"id":"545e447b5222837c2c058ccc","other_names":[],"identifiers":[],"birth_date":null,"death_date":null,"links":[],"contact_details":[],"email":null,"image":null,"created_at":"2015-12-16T14:46:53.841225Z","updated_at":"2015-12-16T14:46:53.841270Z","name":"Datuk Liew Chee Khong @ Liew Chee Choong","family_name":null,"given_name":null,"additional_name":null,"honorific_prefix":null,"honorific_suffix":null,"patronymic_name":null,"sort_name":null,"gender":null,"summary":"","biography":"","national_identity":null,"language_code":"en"},"person_id":"545e447b5222837c2c058ccc","organization":{"id":"5368597219ee29270d8a9fa7","parent_id":null,"founding_date":null,"dissolution_date":null,"links":[],"contact_details":[],"image":"http://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Flag_of_the_Malaysian_Chinese_Association.svg/800px-Flag_of_the_Malaysian_Chinese_Association.svg.png","created_at":"2015-12-17T09:11:56.420472Z","updated_at":"2016-06-29T02:50:42.474634Z","area":null,"name":"Malaysian Chinese Association","classification":"Party","abstract":null,"description":"","language_code":"en"},"organization_id":"5368597219ee29270d8a9fa7","member":null,"member_id":null,"on_behalf_of":null,"on_behalf_of_id":null,"area":null,"area_id":null,"post":{"id":"545e4d5d5222837c2c059890","organization_id":null,"area_id":null,"start_date":null,"end_date":null,"contact_details":[],"links":[],"created_at":"2015-12-22T07:59:33.359962Z","updated_at":"2015-12-22T07:59:33.359997Z","label":"State Assembly Representative","role":null,"language_code":"en"},"post_id":"545e4d5d5222837c2c059890","contact_details":[],"links":[],"start_date":"2004","end_date":"2008","created_at":"2015-12-22T08:54:19.734831Z","updated_at":"2017-03-10T08:12:52.203139Z","label":null,"role":"State Assembly Representative","language_code":"en"}}'

```

post_id is not removed

**Comment by sweemeng** [comment]
to be fixed by #254
